### DISCUSSION TITLE
Commit messages changed format randomly

### DISCUSSION THREAD

**Discussion by naftis** [renovatebot/renovate#29676] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Randomly, our repository's all renovate PR's changed their title and commit messages: https://github.com/opencrvs/opencrvs-core/pull/7086

This is our renovate configuration: https://github.com/opencrvs/opencrvs-core/blob/develop/renovate.json

Why would that happen all of a sudden?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
[Renovate docs, Semantic Commits](https://docs.renovatebot.com/semantic-commits/)

**Reply by naftis** [reply]
We're extending from `"config:recommended"`:
>If you extend from config:recommended then Renovate uses the chore prefix for nearly all updates.

Also it's odd that it randomly just changed without changing configuration

**Reply by rarkins** [reply]
It's not random. Read the first sentence in that link

**Reply by HonkingGoose** [reply, marked as answer]
I'll explain this in a bit more detail for you. :wink: 

Renovate saw the `chore: check a record is already assigned` commit message 2 days ago:

- https://github.com/opencrvs/opencrvs-core/commit/d97656d3bfd3ce49a4e715c31090f0c6966bb060

Renovate afterwards assumes your project is using Semantic Commits. Relevant quote from the docs: [^1]

> Renovate looks at the last 10 commit messages in the base branch to decide if the repository uses semantic commits.
> If there are semantic commits, Renovate uses the [conventional-commits-detector](https://github.com/conventional-changelog/conventional-commits-detector) to decide what convention the commit messages follow.

## How to disable Semantic Commits style PR from Renovate

If you do not want Renovate to automatically guess what commit style you want, put this in your repository's `renovate.json` config file:

```json
{
  "extends": [":semanticCommitsDisabled"]
}
```

See this PR on your repository:

- https://github.com/opencrvs/opencrvs-core/pull/7176

[^1]: [Renovate docs, Semantic Commits](https://docs.renovatebot.com/semantic-commits/)

**Reply by naftis** [reply]
Thank you! I should then explicitly enable them. I might not have been clear but we had semantic commits before and that's the style we want to continue as well. I got confused as to why it changed recently, but we will give the different config settings a try
